### PR TITLE
Gmail single user ga

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44794

## Description
This PR will move GmailSingleUser Integration from beta to GA 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
@bakatzir @anara123 Hi guys, feel free to take a look at the README and let me know your thoughts if you would like to take a look. The problem in the build right now is due to exceeded maximum mailbox size in one of our test emails in one of the test playbooks. Should be handled soon.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
 Additional info available at: https://support.google.com/a/answer/7281227
 
-**All Acount Types:**
+### All Account Types:
 
-* **Optional**: You can use your own Google App instead of the default Demisto App. To create your own app, follow the [Google instructions for Desktop Apps](https://developers.google.com/identity/protocols/OAuth2InstalledApp#prerequisites). 
-
-* Go to the developers credentials page: https://console.developers.google.com/apis/credentials (you may need to setup a new project if you haven't done so in the past).
-* If needed, configure the Consent Screen. Fill in the Consent Screen information you would like to display to your users.
+* **Optional**: You can use your own Google App instead of the default Demisto App. Follow the steps below.
```
Comment (anara123) [inline Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md:50]: this is not optional anymore.
If they are not GSuiteAdmin, their only option is to create their own app

**Review by anara123 — COMMENTED** [review]
looks good

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md (hunk 1 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md]: fixed

File: Packs/GmailSingleUser/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Gmail Single User (Beta)",
```
Comment (Shellyber) [inline Packs/GmailSingleUser/pack_metadata.json:2]: Remove Beta

File: Packs/GmailSingleUser/ReleaseNotes/1_1_5.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Gmail Single User
+- Moved the pack from beta to GA.
```
Comment (Shellyber) [inline Packs/GmailSingleUser/ReleaseNotes/1_1_5.md:4]:
```suggestion
- The integration is now GA.
```

**Review by Shellyber — CHANGES_REQUESTED** [review]
Looks good 👍 
 - Remove the `beta` note from the description file.
 - Check if we need to change the description file accordingly.

File: Packs/GmailSingleUser/pack_metadata.json (hunk 2 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/pack_metadata.json:2]: ack

**Comment by julieschwartz18** [comment]
@[GuyAfik](https://github.com/GuyAfik) - doc review complete.

File: Packs/GmailSingleUser/ReleaseNotes/1_1_5.md (hunk 3 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/ReleaseNotes/1_1_5.md:4]: ack

**Comment by GuyAfik** [comment]
> Looks good 👍
> 
> * Remove the `beta` note from the description file.
> * Check if we need to change the description file accordingly.

ack, changed the description file to a more updated version as well.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md [hunk 4]
```diff
@@ -1,14 +1,62 @@
...
-2. To obtain the **Auth Code** run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account. 
-3. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
-4. Save the instance.
-5. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
-
-## Application Configuration Options
-See the [Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/gmail-single-user) for additional App configuration options:
-* **GSuite Admins:** How to trust the Demisto App.
-* **Bring your own App:** Use your own App instead of the default Demisto App. 
+* To allow Cortex XSOAR to access Gmail, you need to approve the Demisto App (for GSuite Admins) or create your own app (for all other account types).\
```
Comment (Shellyber) [inline Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:3]: Remove trailing `\` in this paragraph

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md [hunk 5]
```diff
@@ -1,14 +1,62 @@
...
+    * To approve the Demisto app, follow the steps in 'GSuite Admins'.\
+    * To create your own app, follow the steps in 'All Account Types'.\
+* Once you have the app, follow the steps in 'Authorization Flow In Cortex XSOAR' to configure OAuth 2.0 authorization in Cortex XSOAR.
```
Comment (Shellyber) [inline Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:6]: Add here the link for `Authorization Flow In Cortex XSOAR`

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md [hunk 6]
```diff
@@ -1,14 +1,62 @@
...
+
+### Authorization Flow In Cortex XSOAR
+1. Create and save an integration instance of the Gmail Single User integration. Do not fill in the *Auth Code* field, this is obtained in the next steps.
+2. To obtain the **Auth Code**, run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account.
+3. If you get a message from Google saying that it cannot verify the application, click **proceed** and click enter for the app name to give the app you created permissions to your Google account. Then click **proceed**.
+4. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
+5. Save the instance.
+6. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
+
+### Fetched Incidents Data
```
Comment (Shellyber) [inline Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:49]: Remove this, not necessary in the description file

**Review by Shellyber — COMMENTED** [review]
- Remove the images from the description file.
 -  Add links to all `Authorization Flow In Cortex XSOAR` steps

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md (hunk 4 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:3]: ack

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md (hunk 6 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:49]: ack

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md (hunk 5 above)
Comment (GuyAfik) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:6]: do you mean a link for the documentation in xsoar-pan-dev? e.g.: https://xsoar.pan.dev/docs/reference/integrations/gmail-single-user#application-authorization-flow

**Comment by GuyAfik** [comment]
> * Remove the images from the description file.
> * Add links to all `Authorization Flow In Cortex XSOAR` steps

1) removed the images.
2) can you elaborate more? links to what exactly?

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md (hunk 5 above)
Comment (Shellyber) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md:6]: An inline link to the right part in the document (same goes for all flows)

**Review by Shellyber — APPROVED** [review]

**Pull request merged**